cmd/minishift/cmd/util/oc.go

```diff
@@ -143,3 +146,29 @@ func GetOcPathForProfile(profileName string) (error, string) {
 	}
 	return nil, instanceCfg.OcPath
 }
+
+// PullOpenshiftImageAndCopyOcBinary pull the openshift image if not available and then copy the client binary to Remote/VM machine
+// Any occurring error is also returned.
+func PullOpenshiftImageAndCopyOcBinary(dockerCommander docker.DockerCommander, requestedOpenShiftVersion string) error {
+	fmt.Printf("-- Pulling the Openshift Container Image ")
+	progressDots := progressdots.New()
+	progressDots.Start()
+	// We need to make sure if images are already exist from the cache then don't pull it again.
+	imageExist, err := dockerCommander.IsImageExist(minishiftConstants.GetOpenshiftImageName(requestedOpenShiftVersion))
+	if !imageExist {
```

Comment:
maybe indicate for `if imageExists` the output as `EXISTS`. it would make our debugging easier.
as no pull happened, no networking was used.